arraycontext/impl/pytato/compile.py

```diff
@@ -90,7 +90,8 @@ def _rec_str(key: Any) -> str:
     return "_".join(_rec_str(key) for key in keys)
 
 
-def _get_arg_id_to_arg_and_arg_id_to_descr(args: Tuple[Any, ...]
+def _get_arg_id_to_arg_and_arg_id_to_descr(args: Tuple[Any, ...],
+                                           kwargs: Mapping[Union[str], Any]
```

Comment:
Why take a union of a single type here? Typo?

Reply:
Whoops, was a leftover from  a previous version. Fixed as a part of  78ff8ff. Thanks for catching that!